### ISSUE TITLE
License Change from LGPL to Apache

### ISSUE THREAD

**Issue by jwvhewitt** [jwvhewitt/gearhead-2#17]
I want to change the license for future releases of GH2 from LGPL to Apache 2.0. This change is necessary for the game to be submitted to Steam. It would only affect future releases; existing versions will remain under LGPL. Before doing so I want to make sure that no contributors have any objections.

**Comment by z9484** [comment]
Did you get special permission from jetrel to release those portraits under LGPL because those two images were originally released under the GPL from Battle For Wesnoth.

And I'm curious what is Steam's problem with the LGPL? Is it because they might statically link your application in a launcher?

**Comment by jwvhewitt** [comment]
No, I did not. I will attempt to contact Jetrel and if either permission is not granted or I can't contact him, I'll remove those portraits immediately. Thanks for bringing this up.

I don't really understand the deal with Steam and LGPL either. Wesnoth itself is on Steam, so I'm curious how they're handling the licensing issues.

**Comment by z9484** [comment]
I did a little more digging and it just looks more confusing. Some people changed their license to LGPL to get on steam, and others changed away from the LGPL so they can get on steam...

Are the other portraits all done by you? I have been wanting to use some in an open source remake of kqlives/kqfork and would like to credit the talented artists(with appropriate license notices).
